Source/Scene/OIT.js

```diff
@@ -86,20 +98,10 @@ function destroyTextures(oit) {
 }
 
 function destroyFramebuffers(oit) {
-  oit._translucentFBO =
-    oit._translucentFBO &&
-    !oit._translucentFBO.isDestroyed() &&
-    oit._translucentFBO.destroy();
-  oit._alphaFBO =
-    oit._alphaFBO && !oit._alphaFBO.isDestroyed() && oit._alphaFBO.destroy();
-  oit._adjustTranslucentFBO =
-    oit._adjustTranslucentFBO &&
-    !oit._adjustTranslucentFBO.isDestroyed() &&
-    oit._adjustTranslucentFBO.destroy();
-  oit._adjustAlphaFBO =
-    oit._adjustAlphaFBO &&
-    !oit._adjustAlphaFBO.isDestroyed() &&
-    oit._adjustAlphaFBO.destroy();
+  oit._translucentFBO.destroyResources();
```

Comment:
Technically this is a slight change from the original code because it now destroys the attachments and the framebuffers.

Reply:
Seems like a good fix to me